src/pybind/mgr/tox.ini

```
@@ -187,3 +188,10 @@ deps =
     jinjaninja
 commands =
     jinja-ninja cephadm/templates
+
+[testenv:gen-mgr-requirements]
```

Since this is in the mgr dir, we can skip the (redundant) reference to the mgr, and what about making it explicit that we're generating the reqs lockfile?:
```suggestion
[testenv:generate-requirements-lockfile]
```